core/src/main/java/pl/ds/howlite/components/models/TitleComponent.java

```diff
@@ -55,6 +55,11 @@ public class TitleComponent implements Styled, Grid {
   @Default(values = "Add your text here")
   private String subtitle;
 
+  @Inject
+  @Getter
+  @Default(values = "Add your anchor-id here")
```

Comment:
For sure we cannot use a default value here. In that case, most of titles will have a "Add your anchor-id here" id.

Reply:
You can use a `@Model(adaptables = Resource.class, defaultInjectionStrategy = OPTIONAL)` as a class annotation